src/org/usfirst/frc/team1024/robot/OI.java

```diff
@@ -14,10 +14,10 @@
  * interface to the commands and command groups that allow control of the robot.
  */
 public class OI {
-	public final Joystick lJoy = new Joystick(0);
+	/*public final Joystick lJoy = new Joystick(0);
 	public final Joystick rJoy = new Joystick(1);
 	
-	
+	*/
 	
 	public OI () {
 		
```

Comment:
Just did this so that it would stop printing to the driverstation that it was unplugged